randovania/games/samus_returns/gui/preset_settings/msr_patches_tab.py

```diff
@@ -29,7 +31,9 @@ def __init__(self, editor: PresetEditor, game_description: GameDescription, wind
         super().__init__(editor, game_description, window_manager)
         self.setupUi(self)
 
-        self.setCentralWidget(self.root_widget)
```

Comment:
Why was the line removed?

Reply:
It didn't seem to make a difference either way.